apps/system/test/unit/lockscreen/widgets/clock/lockscreen_clock_widget_test.js

```diff
@@ -41,12 +40,12 @@ suite('LockScreenClockWidget > ', function() {
       }}
     };
     var method = LockScreenClockWidget.prototype.updateClock;
+    var now = new Date();
     method.call(mockThis);
-    assert.equal(navigator.mozL10n.dummyLocaleFormat(),
+    assert.equal(mockThis.timeFormatter.format(now),
```

Comment:
It make sense. I just get mixed up since I'm reading code in both old and new intentions here, without knowing any Intl things until recent patches (as I bet you of course know nothing about latest LockScreen architecture until this patch). So with your explanation I think it's good now.